content/docs/programs/r-and-r-forms.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "R&R Forms"
+description = "Introduction to R&R Forms"
+date = 2024-08-06T00:00:00+00:00
+updated = 2024-08-06T00:00:00+00:00
+draft = false
+weight = 60
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+lead = "Report and Requisition (R&R) Forms are used to report on item usage and request stock from suppliers. They help to ensure that you have enough stock on hand to meet the needs of your patients."
+toc = true
+top = false
++++
+
+## Setup
+
+To use R&R Forms, ensure that you enable the store preference for `Open mSupply: Uses program module`.
+
+![Store preference](/docs/programs/images/program_pref.png)
+
+You'll also need a program configured and visible in your store. It will need a period schedule associated with it. See the [mSupply docs](https://docs.msupply.org.nz/items:programs) for configuring programs.
```

Comment:
Would be good to link the period / schedule docs. and to save people from adding all the configuration which is on that programs page!
```suggestion
You'll also need a program configured and visible in your store, and the program will need a period schedule associated with it.
Start with configuring [periods and schedules](https://docs.msupply.org.nz/admin:schedules_periods) and then create a program and associate the schedule.
See the [mSupply docs](https://docs.msupply.org.nz/items:programs) for configuring programs - note though that you'll only need the start of this process, which is to create a program and associate a schedule. For R&R forms, the other program configuration aspects aren't used yet.
```